tempus-sdk/utils/Decimal.ts

```diff
@@ -180,6 +203,10 @@ export class Decimal {
     return this.int <= this.toScaledBigInt(x);
   }
 
+  public isZero(): boolean {
+    return this.int === ZERO;
```

Comment:
```suggestion
    return this.int === 0n;
```

Does the extra const provides any benefit?

Reply:
We'll have to upgrade ES in another PR to get that feature